static/pages/partnerships/Partnerships.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppiaFoundationWebsite.controller(
+  'PartnershipsPage', ['$scope', function($scope) {
+    $scope.submitContactUsForm = function(
+        fullName, email, organization, comment) {
+      var requestParams = {
```

Comment:
You want an $http.post to your backend handler here (there should be a main.py etc. for the routing).